docs/career_assessment_how_to.md

```diff
@@ -0,0 +1,65 @@
+# Career assessment
+## How does it work?
+Each "Me" or "Not Me" response is matched with a trait like "Adaptable" or "Organized". Each career is associated with a few traits. When results are calculated, each career is given a score to reflect how closely it matches the traits chosen by the job seeker. The three careers with the highest score are displayed.
+
+Matching an image with a trait can happen three ways:
+1. Positive association: responding "Me" to an image is a "Me" response to the trait (Example: 'Working with customers' -> 'Customer service')
```

Comment:
I had a think about the logic for a second. Do you mean to say that...

selecting the "Me" option to an image matches to a "Me" response trait.